Nitrox.Test/Client/Communication/MultiplayerSession/ConnectionState/DisconnectedStateTests.cs

```diff
@@ -29,7 +29,7 @@ public void NegotiateShouldStartTheClientOnTheContext()
             Disconnected connectionState = new Disconnected();
```

Comment:
Tests are failing for me with this change

Reply:
Tests are working once again and the Async method is properly awaited